docs/src/main/asciidoc/spring-cloud-gateway.adoc

```diff
@@ -1567,6 +1567,19 @@ To retrieve the <<gatewayfilter-factories,GatewayFilter factories>> applied to r
 
 The response contains details of the GatewayFilter factories applied to any particular route. For each factory is provided the string representation of the corresponding object (e.g., `[SecureHeadersGatewayFilterFactory@fceab5d configClass = Object]`). Note that the `null` value is due to an incomplete implementation of the endpoint controller, for that it tries to set the order of the object in the filter chain, which does not apply to a GatewayFilter factory object.
 
+=== Retrieving route predicates
+To retrieve the <<route-predicate-factories,RoutePredicate factories>> applied to routes, make a `GET` request to `/actuator/gateway/routepredicates`. The resulting response is similar to the following:
+
+----
```

Comment:
it would be god to provide the route these are applied to

Reply:
You mean route example that apply PredicateFactory?

Reply:
I mean which route id are these predicates associated with

Reply:
I tried.. but if I get predicates applied each routes from `RouteDefinitionLocator`, It works fine. But **I cannot get route that made with `RouteLocatorBuilder`.**

Because of the above reasons, I tried to get predicates from `RouteLocator`, but I cannot get `Predicates` name from `Route`. So I cannot match `RoutePredicateFactory` with `Route`. 

Is there a way to solve this?

Reply:
I dont see the value in this if we cant associate the predicates with a route

Reply:
Just a list of Predicates is enough?